latest/ug/automode/auto-accelerated.adoc

```diff
@@ -0,0 +1,342 @@
+
+Amazon EKS Auto Mode includes the drivers required for certain instance types, such as NVIDIA and {aws} Neuron drivers. You do not have to install or update these drivers. 
+
+EKS Auto Mode manages the drivers for: 
+
+* link:ai/machine-learning/trainium/[{aws} Trainium, type="marketing"]
+* link:ai/machine-learning/inferentia/[{aws} Inferentia, type="marketing"]
+* NVIDIA GPU link:ec2/latest/instancetypes/ac.html[Amazon EC2 accelerated instances,type="documentation"]
+** Including the https://github.com/NVIDIA/k8s-device-plugin[NVIDIA device plugin for Kubernetes]. EKS Auto Mode includes this plugin, and is not visible as a daemon set within your cluster. 
+
+Amazon EKS Auto Mode also includes support for link:hpc/efa/[Elastic Fabric Adapter,type="marketing"].
+
+//GDC: What does the EFA thing mean?
+
+Amazon EKS Auto Mode eliminates the toil of GPU driver management. 
+
+You can also benefit from cost savings by scaling the cluster to zero. You can configure EKS Auto Mode to terminate instances when no workloads are running. This is useful for batch based inference workloads.
+
+The following provides an example of how to launch GPU-based workloads with Amazon EKS Auto Mode.
+
+== Prerequisites
+
+* A Kubernetes cluster with Amazon EKS Auto Mode configured.
+* A `default` EKS Node class as created when the `general-purpose` or `system` Managed Node Pools are enabled.
+
+== Step 1: Deploy a GPU workload
+
+In this example, you will create a NodePool for NVIDIA inference based workloads that requires 45GB GPU memory. With EKS Auto Mode, you use Kubernetes scheduling constraints to define your instance requirements. 
+
+To deploy the Amazon EKS Auto Mode `NodePool` and the sample `workload`, review the following NodePool and Pod definition and save as `nodepool-gpu.yaml` and `pod.yaml`:
+
+*nodepool-gpu.yaml*
+
+[source,yaml]
+----
+apiVersion: karpenter.sh/v1
+kind: NodePool
+metadata:
+  name: gpu
+spec:
+  disruption:
+    budgets:
+    - nodes: 10%
+    consolidateAfter: 1h
+    consolidationPolicy: WhenEmpty
+  template:
+    metadata: {}
+    spec:
+      nodeClassRef:
+        group: eks.amazonaws.com
+        kind: NodeClass
+        name: default
+      requirements:
+        - key: "karpenter.sh/capacity-type"
+          operator: In
+          values: ["on-demand"]
+        - key: "kubernetes.io/arch"
+          operator: In
+          values: ["amd64"]
+        - key: "eks.amazonaws.com/instance-family"
+          operator: In
+          values:
+          - g6e
+          - g6
+      terminationGracePeriod: 24h0m0s
+----
+
+*pod.yaml*
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nvidia-smi
+spec:
+  nodeSelector:
+    eks.amazonaws.com/instance-gpu-memory: "45776"
```

Comment:
this will only match on g6e above, and given that GPU memory and arch of one GPU architecture is not equivalent to another, its more representative to use the card name itself (as in - "I want this pod to run on L40S GPUs") 

```suggestion
    eks.amazonaws.com/instance-gpu-name: l40s
```